google-cloud-datastore/lib/google/cloud/datastore.rb

```diff
@@ -98,12 +100,13 @@ module Datastore
       #   datastore.save task
       #
       def self.new project_id: nil, credentials: nil, scope: nil, timeout: nil,
-                   client_config: nil, emulator_host: nil, project: nil,
-                   keyfile: nil
+                   client_config: nil, endpoint: nil, emulator_host: nil,
+                   project: nil, keyfile: nil
         project_id    ||= (project || default_project_id)
         scope         ||= configure.scope
         timeout       ||= configure.timeout
         client_config ||= configure.client_config
+        endpoint ||= configure.endpoint
```

Comment:
Can you either make the whitespace match the formatting around it, or remove the fancy formatting in the other lines?

Reply:
Done!